src/store/ChartSettingStore.js

```diff
@@ -177,4 +180,13 @@ export default class ChartSettingStore {
         }, 10);
         this.menu.setOpen(false);
     }
+
+    @action.bound setAutoScale(value) {
+        if (this.isAutoScale === value) { return; }
+
+        this.isAutoScale = value;
+        logEvent(LogCategories.ChartControl, LogActions.ChartSetting, `${value ? 'Show'  : 'Hide'} AutoScale`);
```

Comment:
@morteza-binary here i think it's better if you log something like `Change AutoScale to ${AutoScale}`.